lib/samples/identify_features_in_wms_layer/identify_features_in_wms_layer.dart

```diff
@@ -0,0 +1,191 @@
+    );
+  }
+
+  void onMapViewReady() async {
+    // Create a map with a basemap and set to the map view controller.
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISDarkGrayBase);
+    _mapViewController.arcGISMap = map;
+
+    // Create a URI to a WMS service showing EPA water info.
+    final wmsServiceUri = Uri.parse(
+      'https://watersgeo.epa.gov/arcgis/services/OWPROGRAM/SDWIS_WMERC/MapServer/WMSServer?request=GetCapabilities&service=WMS',
+    );
+    // Create a list of WMS layer names to display.
+    final layerNames = ['4'];
+    // Create a WMS Layer using the service URI and layer names, and load.
+    _wmsLayer = WmsLayer.withUriAndLayerNames(
+      uri: wmsServiceUri,
+      layerNames: layerNames,
+    );
+    await _wmsLayer.load();
+    // Once loaded get the extent of the layer.
+    final layerExtent = _wmsLayer.fullExtent;
+    // Set the viewpoint to the extent of the layer.
+    if (layerExtent != null) {
+      _mapViewController.setViewpoint(Viewpoint.fromTargetExtent(layerExtent));
+    }
+    // Add the WMS Layer to the map's operational layers.
+    map.operationalLayers.add(_wmsLayer);
+    // Set the ready state variable to true to enable the sample UI.
+    setState(() => _ready = true);
+  }
+
+  void onTap(Offset localPosition) async {
+    // When the map view is tapped, perform an identify operation on the WMS layer.
+    final identifyLayerResult = await _mapViewController.identifyLayer(
+      _wmsLayer,
+      screenPoint: localPosition,
+      tolerance: 12.0,
+      maximumResults: 1,
+    );
+
+    // Check if there are any features identified.
+    final features =
+        identifyLayerResult.geoElements.whereType<WmsFeature>().toList();
+    if (features.isNotEmpty) {
+      // Get the identified WMS feature.
+      final identifiedWmsFeature = features.first;
+      // Retrieve the feature's HTML content.
+      final htmlContent = identifiedWmsFeature.attributes['HTML'] as String;
+      // This particular server will produce an identify result with an empty table when there is no identified feature.
+      // This sample checks for the presence of OBJECTID in the HTML, and doesn't display the result if it is missing.
+      if (htmlContent.contains('OBJECTID')) {
+        // Zoom into the HTML content to make it more readable.
+        final zoomedHtmlContent = updateHtmlInitialScale(htmlContent);
+        // Load the HTML content via the web view controller.
+        await _webViewController.loadHtmlString(zoomedHtmlContent);
+        // Configure a dialog that will display the results.
+        showResultsDialog();
+      }
+    }
+  }
+
+  void showResultsDialog() {
+    // Show a dialog containing a web view widget.
+    showDialog(
+      context: context,
+      builder: (context) => AlertDialog(
+        title: Text(
+          'Identify Result',
+          style: Theme.of(context).textTheme.titleMedium,
+        ),
+        content: SizedBox(
+          height: 150,
```

Comment:
does this work in landscape orientation? Otherwise we might need like a BoxConstraints/maxHeight/SingleChildScrollView sort of thing.

Reply:
Yeah I tested quite a bit in landscape as actually it's better viewed in landscape as the whole HTML table width fits. The HTML content height is consistent so it looks the same height wise.